src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.Aot.targets

```diff
@@ -0,0 +1,71 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.Aot.targets
+
+.NET 6 AOT support. You can find "legacy" Xamarin.Android AOT support
+in Xamarin.Android.Legacy.targets.
+
+For <MonoAOTCompiler/> usage, see:
+* https://github.com/dotnet/runtime/blob/15dec9a2aa5a4236d6ba70de2e9c146867b9d2e0/src/tasks/AotCompilerTask/MonoAOTCompiler.cs
+* https://github.com/dotnet/runtime/blob/15dec9a2aa5a4236d6ba70de2e9c146867b9d2e0/src/mono/netcore/nuget/Microsoft.NET.Runtime.MonoAOTCompiler.Task/README.md
+
+***********************************************************************************************
+-->
+<Project>
+
+  <UsingTask
+      Condition=" '$(AotAssemblies)' == 'true' "
+      TaskName="MonoAOTCompiler"
+      AssemblyFile="$(MonoAOTCompilerTaskAssemblyPath)"
+  />
+  <UsingTask TaskName="Xamarin.Android.Tasks.GetAotArguments" AssemblyFile="$(_XamarinAndroidBuildTasksAssembly)" />
+
+  <Target Name="_AndroidAot"
+      Condition=" '$(AotAssemblies)' == 'true' "
+      Inputs="$(_BuildApkEmbedInputs)"
+      Outputs="$(_BuildApkEmbedOutputs)">
+    <!--
+      TODO:
+      * I set every [Input] with a non-existent property.
+      * AotProfilePath is a single string, not sure how to pass @(_AotProfiles)?
+      * Hardcoded microsoft.netcore.app.runtime.aot.osx-x64.cross
+      * Hardcoded '.21' to be replaced with ''
+    -->
+    <GetAotArguments
+        AndroidAotMode="$(AndroidAotMode)"
+        AndroidNdkDirectory="$(_AndroidNdkDirectory)"
+        AndroidBinUtilsDirectory="$(AndroidBinUtilsDirectory)"
+        AndroidApiLevel="$(_AndroidApiLevel)"
+        ManifestFile="$(IntermediateOutputPath)android\AndroidManifest.xml"
+        AndroidSequencePointsMode="$(_SequencePointsMode)"
+        AotAdditionalArguments="$(AndroidAotAdditionalArguments)"
+        AotOutputDirectory="$(_AndroidAotBinDirectory)"
+        RuntimeIdentifier="$(RuntimeIdentifier)"
+        EnableLLVM="$(EnableLLVM)"
```

Comment:
IIRC we ship LLVM only mode

/cc @akoeplinger

Reply:
Turning on `$(EnableLLVM)` does not appear to work quite yet: https://github.com/dotnet/runtime/issues/56386

We could make it the default value when it's working, if it makes sense to do so.